submissions/DaniaB24/html-css-popup/style.css

```diff
@@ -0,0 +1,226 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style-type: none;
+    color: #757575;
+}
+
+body {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    overflow-x: hidden;
+}
+
+.nav-menu {
+    margin-top: 20px;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style-type: none;
+    color: black;
+    width: 100%;
+    height: 20%;
+}
+
+.nav__list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    background-color: transparent;
+    font-size: 16px;
+    padding: 0px 0 5px 0;
+    margin: 20px 0 0 0;
+}
+
+.extra__label {
+    position: absolute;
+    z-index: 1;
+    background: lightblue;
+    font-size: 15px;
+    padding: 15px 0 10px 0;
+    height: 50px;
+    width: 348px;
+    right: 90.5px;
+    top: 520px;
+    text-align: center;
+}
+
+.li_popup {
+    font-family: Arial, Helvetica, sans-serif;
+    list-style-type: none;
+    text-decoration: none;
+    width: 33%;
+    padding: 20px 0;
+    display: flex;
+    justify-content: center;
+}
+
+.nav__link {
+    padding: 2px;
+    margin: 0 0px 0 20px;
+}
+
+.menu-top {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: white;
+    border: 2px solid darkgray;
+    box-shadow: 0 0 15px lightgray;
+    right: 88px;
+    justify-content: center;
+}
+
+.menu-top:focus {
+    border: 2px solid black;
+}
+
+p {
+    width: 50%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.popup {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 40px;
+    margin: 0 auto 8px;
+}
+
+#popup_chat {
```

Comment:
Good practice -- to avoid ID-selectors for styles. ID should be used for JS only. Could you please fix it for all selectors where it was used?